src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTest.java

```diff
@@ -147,6 +149,13 @@ public void should_process_other_zip_files_if_previous_zip_fails_to_process() th
         //This verifies only pdf included in the zip with metadata was processed
         verify(documentManagementService).uploadDocuments(ImmutableList.of(okPdf));
 
+        // Verify if event registered for zipfile processing event
+        List<ProcessEvent> processEvents = processEventRepository.findAll();
+        assertThat(processEvents.stream().map(ProcessEvent::getEvent).collect(toList()))
+            .contains(
+                ZIPFILE_PROCESSING_STARTED
+            );
```

Comment:
probably not relevant in 'should_process_other_zip_files_if_previous_zip_fails_to_process'